code/game/objects/items/GriffeningPorted.dm

```diff
@@ -0,0 +1,805 @@
+			var/choice = href_list["pick"]
+			var/C = new choice(cardUser.loc)
+			src.cards -= choice
+			cardUser.put_in_hands(C)
+			cardUser.visible_message("<span class='notice'>[cardUser] draws a card from [cardUser.p_their()] hand.</span>", "<span class='notice'>You take the card from your hand.</span>")
+
+			interact(cardUser)
+			if(src.cards.len < 3)
+				src.icon_state = "[deckstyle]_hand2"
+			else if(src.cards.len < 4)
+				src.icon_state = "[deckstyle]_hand3"
+			else if(src.cards.len < 5)
+				src.icon_state = "[deckstyle]_hand4"
+			if(src.cards.len == 1)
+				var/A = src.cards[1]
+				qdel(src)
+				cardUser.put_in_hands(A)
+				to_chat(cardUser, "<span class='notice'>You also take [cards[1]] and hold it.</span>")
+				cardUser << browse(null, "window=cardhand")
+				return
+
+/obj/item/griffeningdeck/cardhand/attack_hand(mob/user)
+		user.put_in_hands(src)
+
+/obj/item/griffeningdeck/cardhand/attackby(obj/item/griffeningdeck/cardhandsinglecard/C, mob/living/user, params)
+	if(istype(C))
+		src.currenthand += C
+		user.visible_message("[user] adds a card to [user.p_their()] hand.", "<span class='notice'>You add the [C.cardname] to your hand.</span>")
+		qdel(C)
+		interact(user)
+		if(currenthand.len > 4)
+			src.icon_state = "[deckstyle]_hand5"
+		else if(currenthand.len > 3)
+			src.icon_state = "[deckstyle]_hand4"
+		else if(currenthand.len > 2)
+			src.icon_state = "[deckstyle]_hand3"
+	else
+		return ..()
+
+/obj/item/griffeningdeck/cardhand/single
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "singlecard_nanotrasen_up"
+	w_class = WEIGHT_CLASS_TINY
+	LVL = 0
+	ATK = 0
+	DEF = 0
+	pixel_x = -5
+
+/obj/item/griffeningdeck/cardhand/single/examine(mob/user)
+	to_chat(user, "[ATK] ATK| [DEF] DEF| [LVL] LVL| [desc]")
+
+/obj/item/griffeningdeck/cardhand/single/attack_hand(mob/user)
+	user.put_in_hands(src)
+
+/obj/item/griffeningdeck/cardhand/single/creature/captain
+
+	name = "Captain"
+	desc = "Captain cannot be played if there's a nuclear operative on the field or if there's another captain on the field. Captain can only be played if the area 'bridge' is activated. If the captain is successfully summoned, you may immediately equip a 'energy gun' from your deck."
+	LVL = 7
+	ATK = 60
+	DEF = 40
+
+/obj/item/griffeningdeck/cardhand/single/creature/HeadOfPersonal
+
+	name = "Head Of Personal"
+	desc = "If this card is in play, your opponent cannot attack 'assistant'. Any assistant that is in play and on the owner's side of the field can activate the 'exchange' ability."
+	LVL = 7
+	ATK = 20
+	DEF = 65
+
+/obj/item/griffeningdeck/cardhand/single/creature/Assistant
+
+	name = "Assistant"
+	desc = "Has the ability exchange; if the 'head of personal' is in play and on the owner's side of the field, sacrifice the assistant and immediately summon a non antagonist human that is either level three or below from the hand or deck. If the 'head of security' is in play, you can also use exchange to get a security officer."
```

Comment:
you really need someone to standardize these to something like the current yugioh card format because these are mildly unclear at best